modules/integration-tests/src/test/java/org/wso2/carbon/apimgt/rest/integration/tests/publisher/APILifeCycleTestCaseIT.java

```diff
@@ -178,7 +179,7 @@ public void testApisApiIdCommentsPost() throws AMIntegrationTestException {
         comment.setApiId(apiId);
         comment.setCommentText("this is a sample comment");
         comment.setCategory("sample category");
-        comment.setParentCommentId(UUID.randomUUID().toString());
+        comment.setParentCommentId(null);
```

Comment:
Can you explain why this is set to null?

Reply:
When inserting a new "comment", it does not have a parent comment. When inserting a "reply" for a particular comment, then that reply will have the parent comment id of its parent comment.